src/Providers/ProviderInterface.php

```diff
@@ -14,14 +14,25 @@
      *
      * @author Morten Rugaard <moru@nodes.dk>
      * @date   13-10-2015
-     *
      * @access public
      * @param  string $key
      * @param  array  $replacements
      * @param  string $locale
      * @param  string $platform
      * @return string
      * @throws \Nodes\Translate\Exception\InvalidKeyException
+     * @throws \Nodes\Translate\Exception\MissingCredentialsException
      */
     public function get($key, $replacements = [], $locale = null, $platform = null);
+
+    /**
+     * setApplication
+     *
+     * @author Casper Rasmussen <cr@nodes.dk>
+     * @access public
+     * @param $application
```

Comment:
Missing what type the variable is. Remember it if can be almost everything, like `string`, `array` etc. you should use the `mixed` type instead of going `string|array|integer`.